### PR TITLE
ENT-6165 Updated contrib/masterfiles-stage scripts and instructions to be accurate

### PR DESCRIPTION
**Pull request by craigcomstock**
Hub works as installed but there are a few missing pieces in this core/contrib dir.

### CODE REVIEW

File: contrib/masterfiles-stage/README.org [hunk 1]
```diff
@@ -60,7 +60,7 @@ earlier versions of git may or may not work.
 :ID:       2aeaaa9b-1229-4c14-b130-6d86e370de42
 :END:
 Installation simply consists of saving =masterfiles-stage.sh= and its supporting
-=common.sh= to the same directory on your policy server, making sure that
+=common.sh= and =ssh-wrapper.sh= to the same directory on your policy server, making sure that
```
Comment (nickanderson) [inline contrib/masterfiles-stage/README.org:63]: Did you pull ssh-wrapper.sh from nova or mission-portal?

Comment (craigcomstock) [reply contrib/masterfiles-stage/README.org:63]: there is only one in nova AFAICT

https://github.com/cfengine/nova/tree/master/api/http/dc-scripts

Can't find one in mission-portal.

Comment (craigcomstock) [reply contrib/masterfiles-stage/README.org:63]: may be more files we need to pull/sync. Will re-test using these current docs and adjust. :+1:

Comment (craigcomstock) [reply contrib/masterfiles-stage/README.org:63]: yes, in order to satisfy the examples we also need `git-askpass.sh`

**Review by nickanderson — APPROVED** [review]

**Pull request merged**